src/main/java/blockchain/LightChainNode.java

```diff
@@ -39,6 +41,9 @@
 	private SimLog simLog = new SimLog(true);
 	private Logger logger;
 	Parameters params;
+	public int ownerToken;
+	public boolean testt;
```

Comment:
These variables should have a comment describing what they store. Also, `testt` is not a self-explanatory name for the main code variable.